examples/unix/c11/z_get.c

```diff
@@ -112,6 +130,14 @@ int main(int argc, char **argv) {
         printf("Unable to send query.\n");
         return -1;
     }
+    struct timespec query_timeout;
+    clock_gettime(CLOCK_REALTIME, &query_timeout);
```

Comment:
This should use the public timer mechanisms found in `include/zenoh-pico/system/platform.h`, this impacts all the following clock calls